mcs/build/profiles/winaot.make

```diff
@@ -28,6 +28,8 @@ PROFILE_MCS_FLAGS = \
 	-d:DISABLE_REMOTING \
 	-d:DISABLE_COM \
 	-d:WIN_PLATFORM \
+	-d:FEATURE_CONSOLE \
+	-d:NO_MULTIPLE_APPDOMAINS \
```

Comment:
this is wrong, it needs to be set in the makefile as `NO_MULTIPLE_APPDOMAINS=1` (check monotouch_tv.make).